app/views/application/_categories.html.haml

```diff
@@ -1,7 +1,7 @@
 - categories1, categories2 = categories[0...7], categories[7..-1]
 %ul.menu__items.menu__items--first
   - categories1.each do |category|
-    %li.menu__item{class: "menu__item--#{category.slug}", data: {menu_item_state: category.root_of?(current_category) ? 'current' : nil}}
+    %li.menu__item{class: "menu__item--category__#{category.slug}", data: {menu_item_state: category.root_of?(current_category) ? 'current' : nil}}
```

Comment:
How about "menu__item--category-#{category.slug}" ?
In [BEM](http://getbem.com/naming/), `__` means element.
It should be separated by `-` I think.